x/valset-pref/client/query_proto_wrap.go

```diff
@@ -0,0 +1,22 @@
+package client
+
+import (
+	"context"
+
+	validatorprefkeeper "github.com/osmosis-labs/osmosis/v12/x/valset-pref"
+	"github.com/osmosis-labs/osmosis/v12/x/valset-pref/client/queryproto"
+)
+
+type Querier struct {
+	K validatorprefkeeper.Keeper
+}
+
+var _ queryproto.QueryServer = Querier{}
+
+func NewQuerier(k validatorprefkeeper.Keeper) Querier {
+	return Querier{K: k}
+}
+
+func (q Querier) UserValidatorPreferences(ctx context.Context, req *queryproto.QueryUserValidatorPreferences) (*queryproto.QueryUserValidatorPreferenceResponse, error) {
```

Comment:
This is currently a stub. Do we have an issue that tracks this being completed?

Reply:
Yup https://github.com/osmosis-labs/osmosis/issues/3204